app/adapters/alma_adapter.rb

```diff
@@ -67,10 +67,38 @@ def format_holding(holding_items_array)
     {
       "holding_id" => holding_items_array.first.holding_data["holding_id"],
       "call_number" => holding_items_array.first.holding_data["call_number"],
-      "items" => holding_items_array.map { |bib_item| bib_item.item["item_data"] }
+      "items" => holding_items_array.each_with_index.map { |bib_item, idx| format_item(bib_item, idx) }
     }
   end
 
+  def format_item(bib_item, idx)
+    bib_item.item["item_data"].merge(
+      "id" => bib_item.item["item_data"]["pid"],
+      "copy_number" => bib_item.item["holding_data"]["copy_id"].to_i,
+      "item_sequence_number" => idx + 1,
+      "temp_location" => temp_location(bib_item),
```

Comment:
I think we should remove the item_sequence_number based on https://github.com/pulibrary/marc_liberation/issues/845#issuecomment-709450744. @mzelesky can you check?

Reply:
There is no item sequence number in Alma. Until we have a full dump, I won't know if the sequence of items returned in the publishing job is significant (meaning that the order is intentional). We may need to sort items by enumeration.